pkg/api/v1/exposure_types.go

```diff
@@ -68,9 +68,7 @@ const (
 
 // Publish represents the body of the PublishInfectedIds API call.
 // temporaryExposureKeys: Required and must have length >= 1 and <= 21 (`maxKeysPerPublish`)
-// healthAuthorityID: The identifier for the mobile application.
-//  - Android: The App Package AppPackageName
```

Comment:
Did you intend to remove `Android` and `IOS` or should we put those somewhere?

Reply:
/lgtm /hold

Reply:
/lgtm /hold

Reply:
yes, that was leftover from v1alpha1